iocore/net/SSLSessionTicket.cc

```diff
@@ -100,12 +101,20 @@ ssl_callback_session_ticket(SSL *ssl, unsigned char *keyname, unsigned char *iv,
         }
 
         netvc.setSSLSessionCacheHit(true);
+
+#if TS_HAS_TLS_EARLY_DATA
+        if (SSL_version(ssl) == TLS1_3_VERSION && config->server_max_early_data > 0) {
```

Comment:
This will break with TLS 1.4, should use `>=`.  There are other checks like this.

Reply:
fixed